index.html

```diff
@@ -682,18 +682,24 @@ <h4>Registration Information</h4>
                 </section>
                 <section id="exploration-directory-anonymous-td" class="normative">
                     <h4>Anonymous TD Identifiers</h4>
-                    <a>Anonymous TD</a>s have no `id` (or `@id`) and are considered as blank nodes [[JSON-LD11]].
-                    The server assigns a local identifier to such TDs to allow management and retrieval
-                    from the directory.
+                    <a>Anonymous TD</a>s are considered as JSON-LD blank nodes [[JSON-LD11]].
+                    The server assigns a local identifier to such TDs
+                    to enable management and retrieval from the directory.
                     <span class="rfc2119-assertion" id="tdd-reg-anonymous-td-identifier">
                         In situations where the server exposes an Anonymous TD, it MUST
-                        add the local identifier to the Anonymous TD as a blank
+                        add the local identifier to the TD as a blank
                         node identifier to allow local referencing.
                     </span>
                     <a href="https://json-ld.org/spec/latest/json-ld/#dfn-blank-node-identifiers">Blank node identifiers</a>
-                    begin with `_:`. For example,
-                    a TD that has local identifier `48951ff3-4019-4e67-b217-dbbf011873dc`
-                    will have the following blank node identifier: `_:48951ff3-4019-4e67-b217-dbbf011873dc`.
+                    begin with `_:` which look like IRIs with an underscore scheme. 
+                    <p>
+                        For example,
+                        an <a>Anonymous TD</a> that has local identifier `48951ff3-4019-4e67-b217-dbbf011873dc`
+                        will have the following blank node identifier
+                        set as its `id` attribute: `_:48951ff3-4019-4e67-b217-dbbf011873dc`.
+                        <a>Anonymous TD</a>s that embed blank node identifiers are in
```

Comment:
This is still a little ambiguous.  It should say the id is ONLY embedded if an enriched form is requested.  TD should not be modified from their original form unless there is an explicit request to "enrich" them.